configuration/log4j2.xml

```diff
@@ -56,6 +56,7 @@
 		<Logger name="org.openmrs" level="WARN" />
 		<!-- Controls logging of all Service method access: INFO = setters, DEBUG = getters and timing -->
 		<Logger name="org.openmrs.api" level="WARN" />
+		<Logger name="org.openmrs.web.Listener" level="INFO"/>
```

Comment:
by explicitly adding this logger, the buggy code will change this log level to "INFO", instead of "org.openmrs"